packages/core/src/components/expanding-search/expanding-search.scss

```diff
@@ -127,3 +124,15 @@ $container-width: 11.5625rem;
     opacity: 1;
   }
 }
+
+:host(.right-position) {
+  width: $container-width !important;
+}
+
+:host(.right-position.fullWidth) {
+  width: 100% !important;
+
+  .fullWidth {
```

Comment:
nested fullWidth selector on purpose?

Reply:
Yes didnt refactored the component it all:

Host class -> https://github.com/siemens/ix/blob/d6fec805f2c37167d54e0403fcc26f0b752fe696/packages/core/src/components/expanding-search/expanding-search.tsx#L105

Nested class -> https://github.com/siemens/ix/blob/d6fec805f2c37167d54e0403fcc26f0b752fe696/packages/core/src/components/expanding-search/expanding-search.tsx#L126